src/Core/System/Snippet/SnippetValidator.php

```diff
@@ -20,12 +25,23 @@ public function __construct(
     }
 
     /**
-     * @return array<string, mixed>
+     * @deprecated tag:v6.8.0 - reason:return-type-change - Will return SnippetValidationStruct
```

Comment:
I know, that you probably want to stick with the well-named method, but this not good to handle for people to make the usage of this class forward compatible. 

we should have an old method, that is deprecated and thrown away and a new method which should be used from now on. 

keeping a method and changing the return type between major versions is not good

Reply:
we might want to get rid of the interface, too :grimacing: as you would need a new interface with the new method, to make it correct

Reply:
kay, deprecated it